python/python/glide/async_commands/cluster_commands.py

```diff
@@ -415,6 +415,43 @@ async def function_delete(
             ),
         )
 
+    async def fcall_route(
```

Comment:
There's an implementation of fcall already in core.py, so because there's no overloading in Python, we had to use a different method name here. I was originally going to use a Union of keys and route, but Yi-pin already merged his implementation with fcall_ro_route, so I just aligned with his API choice.